autoload/neomake/makers/ft/sml.vim

```diff
@@ -0,0 +1,29 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#sml#EnabledMakers()
```

Comment:
This function can simply be:

```
function! neomake#makers#ft#sml#EnabledMakers()
    return ['smlnj']
endfunction
```

Before we try to use a maker, `neomake#utils#Exists` is called on its `exe` property. We only need to check if things exist directly in functions like this when there are special requirements (e.g. only enable maker x if maker y is not present).